### PR TITLE
Polish translation

### PR DESCRIPTION
**Pull request by bla-ke**
# Polish translation
Polish translation. It still needs some work, but it's pretty useful and easy to understand. If I find a mistake, I will correct it.

Regards, Daria

### CODE REVIEW

**Comment by liudongmiao** [comment]
I don't understand your language. 
So, if you have finished, please tell me.

**Comment by bla-ke** [comment]
Hello!
It's ready to be published in Brevent. Checked everything, but if I find a mistake while using your application, I will change it.

Greetings, Daria

File: brevent/src/main/res/values-pl/strings.xml [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+    <string name="brevent_allow_root">Uruchom jako ROOT</string>
+    <string name="brevent_allow_root_label">Pozwala ROOT-owi na uruchomienie serwera Brevent\n\n Wymagane 3 Brefoile dla wersji Play</string>
+    <string name="brevent_allow_root_label_debug">Pozwala ROOT-owi na uruchomienie serwera Brevent bez \"Debugowania USB\"</string>
+
+    <string name="show_donation">Pokaż płatności</string>
+    <string name="show_donation_summary_not_play" />
+    <string name="show_donation_play">Dzięki za kupno %s Brefoil w Play</string>
+    <string name="show_donation_rmb">Dzięki za kupno %s Brefoil</string>
+    <string name="show_donation_contributor">Dzięki za bezcenny wkład w Brevent</string>
+    <string name="show_donation_play_and_rmb">Dzięki za kupno %s Brefoil w Play, %2$s Brefoil poprzez inne źródło</string>
```
Comment (liudongmiao) [inline brevent/src/main/res/values-pl/strings.xml]: The first `%s` should be `%1$s`.

File: brevent/src/main/res/values-pl/strings.xml [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+    <string name="show_donation_play_and_contributor">Dzięki za kupno %s Brefoil w Play i bezcenny wkład w Brevent</string>
+    <string name="show_donation_rmb_and_contributor">Dzięki za kupno %s Brefoil i bezcenny wkład w Brevent</string>
+    <string name="show_donation_play_and_rmb_and_contributor"> Dzięki za kupno %s Brefoil w Play, %2$s Brefoil poprzez inne źródło i bezcenny wkład w Brevent</string>
```
Comment (liudongmiao) [inline brevent/src/main/res/values-pl/strings.xml]: The first `%s` should be `%1$s`.

File: brevent/src/main/res/values-pl-v26/strings.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <string name="brevent_about_developer_adb">Nie wyłączaj \"Debugowania USB\" oraz \"Trybu USB\"</string>
```
Comment (liudongmiao) [inline brevent/src/main/res/values-pl-v26/strings.xml]: `Trybu USB` -> `konfigurację USB`, as in Development Options.

File: brevent/src/main/res/values-pl/strings.xml [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+    <string name="fragment_guide_welcome_message2">Kochamy działanie systemu Android, lecz niektóre aplikacje na tym żerują. Pozostają w tle, drenując baterię, niektóre z nich nawet wyświetlają aktywne powiadomienia, aby przykuć naszą uwagę.\n\nA Brevent ma na zadanie ochronić Twój telefon przed tymi aplikacjami.\nUsuń je z Ostatnich Aplikacji lub dotknij \"Cofnij\", aby zostało wymuszone ich zatrzymanie.\nDotknij \"Home\", aby aplikacje pozostały w tle w celu ukończenia wyznaczonych zadań. Jeżeli przekroczyły ustawiony limit czasu w tle, zostanie wymuszone ich zatrzymanie.</string>
+
+    <string name="fragment_guide_permission">Uprawnienia</string>
+    <string name="fragment_guide_permission_message">Co potrzebujesz? \nProstego polecenia do uruchomienia poprzez ADB.</string>
+    <string name="fragment_guide_permission_message2">Brevent nie będzie działał, jeżeli urządzenie wspierane przez Android się restartuje. Nie będzie również działał na systemie Android O, jeżeli wyłączysz opcję \"Debugowania USB\" lub przełączysz \"Tryb USB\".</string>
```
Comment (liudongmiao) [inline brevent/src/main/res/values-pl/strings.xml]: `Trybu USB` -> `konfigurację USB`, as in Development Options.

**Review by liudongmiao — CHANGES_REQUESTED** [review]

File: brevent/src/main/res/values-pl-v26/strings.xml (hunk 3 above)
Comment (bla-ke) [reply brevent/src/main/res/values-pl-v26/strings.xml]: Corrected and checked, thank you.

**Review by bla-ke — COMMENTED** [review]
Checked, compared with other files and updated.

File: DEVELOP.md [hunk 5]
```diff
@@ -1,3 +1,5 @@
+## English
```
Comment (liudongmiao) [inline DEVELOP.md]: don't translate. it's for developer.

File: CONTRIBUTORS.md [hunk 6]
```diff
@@ -6,11 +6,14 @@
 ## English
 - `DKOUT`: Limber Law
 - `Aviraxp`: Wang Han
-- `liyufan`: Li Yufan
+- `liyufan`: Li Yufan
```
Comment (liudongmiao) [inline CONTRIBUTORS.md]: remove the tail space.

File: DEVELOP.md [hunk 7]
```diff
@@ -31,3 +33,39 @@ apksigner sign --key testkey.pk8 --cert testkey.x509.pem ce.apk
 ```
 
 You can find `testkey.pk8` and `testkey.x509.pem` [here](https://github.com/android/platform_build/blob/master/target/product/security/).
+
```
Comment (liudongmiao) [inline DEVELOP.md]: don't translate. it's for developer.

File: README.md [hunk 8]
```diff
@@ -37,3 +37,25 @@ Currently, default name and package name is `Nrevent`, means `Not Brevent`.
 但是，如果您改动代码，请换个名字，当然也不能叫黒域；同时，2.1.6 以后，包名也须修改。
 
 当前，默认名字为“白域”，包名为“nrevent”（参见英文）。
+
```
Comment (liudongmiao) [inline README.md]: don't translate. it's for developer.

**Review by liudongmiao — CHANGES_REQUESTED** [review]

**Review by bla-ke — COMMENTED** [review]
Corrected.

**Pull request merged**